bazel/repository_locations.bzl

```diff
@@ -105,12 +105,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Abseil",
         project_desc = "Open source collection of C++ libraries drawn from the most fundamental pieces of Google’s internal codebase",
         project_url = "https://abseil.io/",
-        version = "5d8fc9192245f0ea67094af57399d7931d6bd53f",
-        sha256 = "e3812f256dd7347a33bf9d93a950cf356c61c0596842ff07d8154cd415145d83",
+        version = "17c954d90d5661e27db8fc5f086085690a8372d9",
+        sha256 = "2e4ace2ed32a4ccfd29e856ad72b4fd1eae2ec060d3ba8646857fa170d6e8269",
         strip_prefix = "abseil-cpp-{version}",
         urls = ["https://github.com/abseil/abseil-cpp/archive/{version}.tar.gz"],
         use_category = ["dataplane_core", "controlplane"],
-        release_date = "2020-11-24",
+        release_date = "2021-06-07",
```

Comment:
UTC for this commit is `2021-06-03` and this is why CI has failed. As per prior conversation it would be preferable if we could use the latest release of abseil at https://github.com/abseil/abseil-cpp/releases/tag/20210324.2. There is a parallel PR to bump abseil at https://github.com/envoyproxy/envoy/pull/16681

Reply:
@htuch @moderation 

I see comments about https://github.com/envoyproxy/envoy/pull/16681 having trouble compiling under Windows.  Given all these difficulties, I would strongly prefer to go with a patch to grpc and decouple the abseil version bump from the grpc version bump.

Can I please go back to the version with the patch?